lang/src/org/partiql/lang/planner/transforms/Util.kt

```diff
@@ -0,0 +1,13 @@
+
+package org.partiql.lang.planner.transforms
+
+/**
+ * This is the version number of the logical and physical plans supported by this version of PartiQL.
+ *
+ * It would be nice to embed this in the PIG domain somehow, but this isn't supported, so we have to include it
```

Comment:
Didn't we introduce the version number in the new domains. I'm sure you feel the same way, but this is not great. Even if going with this hard-coding, the version doesn't tell anything about backward compatibility; shouldn't we adopt a format like `SemVer`?

Reply:
I really think this aught to be something solved by PIG actually so I created https://github.com/partiql/partiql-ir-generator/issues/121

Reply:
I'll add a link to the ticket.

Reply:
Do you also mind adding some comments to 
```
            (record plan
                (stmt statement)
                (version int)
            )
```
in the `partiql.ion` file, to state that the version number here will be removed and assigned to PIG domains once https://github.com/partiql/partiql-ir-generator/issues/121 is resolved?

Reply:
> I would like to leave this as-is for now if you don't mind.

I think this is something that we need to fix before we expose the API; once we have the version as is and the API is exposed, then it's too late. Happy to discuss further offline.

Reply:
Offline discussion indicates this doesn't need to be addressed in this PR, but should be addressed before this hits `main`.  Creating an issue in the related milestone to track this.

Reply:
Just an update on this; as a comprise, requested having the version as `SemVer` hard-coded for now. @dlurton had a point on using `patch`—which is for bug fixes—might not be applicable to our use-case. We ultimately agreed on having `major` and `minor` notions in the version for now.